setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(
+    name='eedl',
+    version='1.0.0',
+    packages=['tests', 'ee_download', 'ee_download..pytest_cache'],
```

Comment:
Just a note that we'll need to fix the name ee_download here when we change it.

The other two items shouldn't need to be included here though?